Author: Mattias Ellert <mattias.ellert@fysast.uu.se>
Subject: reorder latex packages to avoid bogus syntax check
Bug-Debian: http://bugs.debian.org/750536
Forwarded: https://github.com/doxygen/doxygen/pull/178
Last-Update: 2014-06-04

The latest texlive version adds syntax checking to the float command. This
check is confused if the float package is included before the fixltx2e package,
as is done in the doxygen generated sources.

Index: doxygen/src/latexgen.cpp
===================================================================
--- doxygen.orig/src/latexgen.cpp	2014-06-04 15:25:43.000000000 +0200
+++ doxygen/src/latexgen.cpp	2014-06-04 15:25:43.000000000 +0200
@@ -286,6 +286,7 @@
 
   // Load required packages
   t << "% Packages required by doxygen\n"
+       "\\usepackage{fixltx2e}\n" // for \textsubscript
        "\\usepackage{calc}\n"
        "\\usepackage{doxygen}\n"
        "\\usepackage{graphicx}\n"
@@ -293,7 +294,6 @@
        "\\usepackage{makeidx}\n"
        "\\usepackage{multicol}\n"
        "\\usepackage{multirow}\n"
-       "\\usepackage{fixltx2e}\n" // for \textsubscript
        "\\PassOptionsToPackage{warn}{textcomp}\n"
        "\\usepackage{textcomp}\n"
        "\\usepackage[nointegrals]{wasysym}\n"
